flagset.go

```diff
@@ -119,6 +119,7 @@ func (f *FlagSetFiller) processField(flagSet *flag.FlagSet, fieldRef interface{}
 		}
 	}
 
+	alias := tag.Get("aliase")
```

Comment:
Originally noting a typo/spelling; however, it got me thinking that one or more aliases would be nice to future proof and only slightly more to implement:

```suggestion
	aliases := tag.Get("aliases")
```

Here, `aliases` will just be a `string` and get split at the end.

Reply:
...or maybe it should get split here and let the repetitive code be slightly simpler?

Reply:
whooops, indeed it was an typo